src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,138 @@
+body {
+  margin: 0;
+}
+
+.catalog {
+  --column-count: 1;
+
+  display: grid;
+  padding: 50px 40px;
+  gap: 46px 48px;
+  justify-content: center;
+  grid-template-columns: repeat(var(--column-count), 200px);
+}
+
+@media (min-width: 488px) {
+  .catalog {
+    --column-count: 2;
+  }
+}
+
+@media (min-width: 768px) {
+  .catalog {
+    --column-count: 3;
+  }
+}
+
+@media (min-width: 1024px) {
+  .catalog {
+    --column-count: 4;
+  }
+}
+
+.card {
+  font-family: Roboto, sans-serif;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  border: 1px solid $gray-elements;
+  border-radius: 5px;
+  position: relative;
+  box-sizing: border-box;
```

Comment:
use this style for * selector

Reply:
Not fixed 
<img width="734" alt="image" src="https://github.com/mate-academy/layout_catalog/assets/64750958/9ffe6efd-7558-45dd-9ad8-32468b5a81fb">

Reply:
Not fixed